src/libs/actions/Report.js

```diff
@@ -771,7 +773,148 @@ function readOldestAction(reportID, reportActionID) {
                     onyxMethod: Onyx.METHOD.MERGE,
                     key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
                     value: {
-                        isLoadingMoreReportActions: true,
+                        isLoadingOlderReportActions: true,
```

Comment:
I think this is a property of `REPORT_METADATA` now, is it safe to remove it from the `REPORT` Onyx object?

Reply:
Yes, I think we can move `isLoadingOlderReportActions`, `isLoadingNewerReportActions`, `isLoadingInitialReportActions` from the Report. I'll check the codebase and do this